### PR TITLE
implemented norm01 option for Normalize class

### PR DESCRIPTION
**Pull request by jens-maus**
This PR adds changes to allow to normalize data between [0,1] and not only [-1,1]. In addition, if min and max are equal, value range will now be set to a low value for the normalization not to run into a divide by zero situation anymore.

### CODE REVIEW

File: pytorch3dunet/augment/transforms.py [hunk 1]
```diff
@@ -549,14 +549,26 @@ class Normalize:
...
+        if min_value is not None and max_value is not None:
+          assert max_value > min_value
         self.min_value = min_value
-        self.value_range = max_value - min_value
+        self.max_value = max_value
+        self.norm01 = norm01
 
     def __call__(self, m):
+        if self.min_value is None:
+          self.min_value = np.min(m)
```
Comment (wolny) [inline pytorch3dunet/augment/transforms.py:561]: This implementation has errors:
1. It changes the min/max values in the 1st invocation, so subsequent invocation of this transformer will have incorrect min/max values
2. It makes it not thread-safe which will fail with multi-threaded data loading scenario

**Review by wolny — CHANGES_REQUESTED** [review]
Hi Jens, thanks for your PR. Since your implementation has errors, it cannot be merged in the current form. You can fix the lines where you change the state to be stateless and it should be good to :rocket:

File: pytorch3dunet/augment/transforms.py (hunk 1 above)
Comment (jens-maus) [reply pytorch3dunet/augment/transforms.py]: Thanks for your review. Please see if my latest changes fixes the issues and is enough to be merged.

**Review by wolny — APPROVED** [review]
yes, that looks good thx!

**Pull request merged**